api/docker-compose-base.yml

```diff
@@ -0,0 +1,34 @@
+version: '2.1'
+
+services:
+  api:
+    build:
+      context: ..
+      dockerfile: api/docker/dev/Dockerfile
+    image: bisq/api
+    environment:
+      - LOG_LEVEL=debug
+      - USE_LOCALHOST_FOR_P2P=true
+      - USE_DEV_PRIVILEGE_KEYS=true
+      - SEED_NODES=seed:8000
+      - BTC_NODES=bitcoin:18444
+      - BASE_CURRENCY_NETWORK=BTC_REGTEST
+      - BITCOIN_REGTEST_HOST=NONE
+      - HTTP_API_HOST=0.0.0.0
+      - ENABLE_HTTP_API_EXPERIMENTAL_FEATURES=true
+  seed:
+    image: bisq/seednode
+    environment:
+      - MY_ADDRESS=seed:8000
+      - SEED_NODES=seed:8000
+      - BTC_NODES=bitcoin:18444
+      - USE_LOCALHOST_FOR_P2P=true
+      - BASE_CURRENCY_NETWORK=BTC_REGTEST
+      - BITCOIN_REGTEST_HOST=NONE
+  bitcoin:
+    image: kylemanna/bitcoind:latest
```

Comment:
why does this use a trusted third party's bitcoind image? shouldn't it simply use the default bisq btcnodes?

Reply:
Good catch, but from what I remember the `bisq/bitcoind` is just my tag for `kylemanna/bitcoind`, but it has not been updated since 23 months, while kylemanna updated in 8 months.

Reply:
Why do you have this at all tho, Bitcoin is not part of Bisq, you should just accept an environment variable to specify the URI of the Bitcoin P2P node

Reply:
Developer needs to be able to control bitcoin node during tests and development, i.e. generate new blocks and transfer funds. This all is meant for regtest mode.

Reply:
Ah I see. Well can we maintain our own fork of it instead of trusting a third party?

Reply:
Already did that with commit 51aff9a